src/components/pages/dashboard-pages/ClassForm.jsx

```diff
@@ -128,19 +129,20 @@ const ClassForm = ({
                 Maximum
               </FormInput>
             </div>
+          </div>
+          <div className="lg:w-1/2">
             {formErrors.type && (
               <p className="text-red text-sm font-spartan">{formErrors.type}</p>
             )}
-            <div className="flex gap-4 my-4">
+            <div className="flex gap-4">
               <p className="w-2/5">Select the type of the class:</p>
-              <div className="w-1/2 flex justify-around">
+              <div className="w-1/2 flex gap-8">
                 <label className="flex gap-1 items-center">
```

Comment:
here the sane issue as in PR 38 - missing for attribute to connect with corresponding input